CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## master / unreleased
 
+* [ENHANCEMENT Ruler: Added the following metrics: #2786
+  * `cortex_prometheus_notifications_latency_seconds`
+  * `cortex_prometheus_notifications_errors_total`
+  * `cortex_prometheus_notifications_sent_total`
+  * `cortex_prometheus_notifications_dropped_total`
+  * `cortex_prometheus_notifications_queue_length`
+  * `cortex_prometheus_notifications_queue_capacity`
+  * `cortex_prometheus_notifications_prometheus_notifications_alertmanagers_discovered`
```

Comment:
The metric is registered like this:
```
alertmanagersDiscovered: prometheus.NewGaugeFunc(prometheus.GaugeOpts{
			Name: "prometheus_notifications_alertmanagers_discovered",
			Help: "The number of alertmanagers discovered and active.",
		}, alertmanagersDiscovered),
```

So we should fix this.

```suggestion
  * `cortex_prometheus_notifications_alertmanagers_discovered`
```

Reply:
That last rebase really messed me up 😅